config/clusters/carbonplan/common.values.yaml

```diff
@@ -32,6 +32,9 @@ basehub:
             name: Carbon Plan
             url: https://carbonplan.org
     singleuser:
+      # Temporarily override startTimeout to see if can diagnose scheduling error
+      # https://github.com/2i2c-org/infrastructure/issues/2271
+      startTimeout: 600
```

Comment:
This is the default via basehub chart's values already and becomes redundant.

Reply:
I don't think that's working as we intend as using helm to dump the full manifest and values doesn't have that config override in, and looking at the rendered config file on the server does not include the timout settings